src/main/java/bookface/logic/commands/add/AddBookCommand.java

```diff
@@ -0,0 +1,46 @@
+package bookface.logic.commands.add;
+
+import static bookface.logic.parser.CliSyntax.PREFIX_AUTHOR;
+import static bookface.logic.parser.CliSyntax.PREFIX_TITLE;
```

Comment:
Are these prefixes created in the Clisyntax class?

Reply:
yup!